.travis.yml

```diff
@@ -57,3 +74,11 @@ matrix:
       compiler: gcc
     - <<: *EXTENDED
       compiler: clang
+    - <<: *OSX
+      os: osx
+      before_script:
+        - brew update
+        - HOMEBREW_NO_AUTO_UPDATE=1 brew install cmake jpeg libpng qt5 glew libav libdc1394 imagemagick@6 || echo "installing dependencies returned non-zero!"
```

Comment:
Do you really want to continue in this case?

Reply:
Its a problem which homebrew which returns non-zero in many cases (according to SO) such as failed linking, up-to-date or already installed packages (cannot be upgraded) or otherwise not applicable updates. Problem is that opencv will pull numpy which cannot be [linked](https://travis-ci.org/iclcv/icl/jobs/290129103#L1822) because of an already existing file in /usr/local. For the test build (which does not use the opencv python variant) it does not matter much though. I will however 'simplify' the script by using the formulas again. Let's see if opencv can still be poured or has to be build from source (because of the ffmpeg flag which seem to have been ignored so far). If it has to be built, this will increase build time significantly.